akka-projection-r2dbc/src/it/scala/akka/projection/r2dbc/R2dbcProjectionSpec.scala

```diff
@@ -148,7 +150,13 @@ object R2dbcProjectionSpec {
 
       R2dbcExecutor
         .updateOneInTx(stmt)
```

Comment:
not that it matter for this problem, but these tests should use `R2bcSession.updateOne` and `R2dbcSession.selectOne` instead of the internal api of R2dbcExecutor

(R2dbcSession didn't exist when the test was written)